crawl-ref/source/dat/strings/de/messages.txt

```diff
@@ -1646,16 +1646,16 @@ Dein dämonischer Wächter vermeidet deinen Angriff.
 %s flackert außer Sicht!
 %%%%
 %s flickers out of sight for a moment!
-%s flackert außer Sicht für einen Moment!
+%s flackert für einen Moment außer Sicht!
 %%%%
 %s is dispelled
-%s wird vertrieben
+%s wird gebannt
 %%%%
 %s mind is blasted
-%s Verstand wird zerschlagt
+{gen}%s Verstand wird gesprengt
```

Comment:
Again, the parameter is already possessive.
It should be "Der Verstand %s wird gesprengt" because %s will be replaced by "des Orks", "von Sigmund", etc.